### PR TITLE
Update swap form design

### PR DESCRIPTION
**Pull request by vyorkin**

### CODE REVIEW

**Comment by zerts** [comment]
Let's hide border in the empty state 
Zerts Screenshot 2024-07-25 at 16 54@2x

**Comment by zerts** [comment]
Shouldn't there be different text for zero fees?
Zerts Screenshot 2024-07-25 at 16 55@2x

File: src/ui/pages/SwapForm/SwapForm.tsx [hunk 1]
```diff
@@ -679,35 +679,48 @@ export function SwapFormComponent() {
...
+        <VStack
+          gap={4}
+          style={
+            quotesData.quote || quotesData.isLoading || quotesData.error
+              ? {
+                  borderRadius: 12,
+                  border: '2px solid var(--neutral-200)',
+                  padding: '12px 16px',
+                }
+              : {}
```
Comment (zerts) [inline src/ui/pages/SwapForm/SwapForm.tsx:691]: Can be just `undefined`

**Review by zerts — APPROVED** [review]

File: src/ui/pages/SwapForm/SwapForm.tsx (hunk 1 above)
Comment (vyorkin) [reply src/ui/pages/SwapForm/SwapForm.tsx:691]: `undefined` is better, thanks!

**Review by everdimension — APPROVED** [review]

**Pull request merged**